### PR TITLE
Feat/provide store data from php

### PR DESCRIPTION
**Pull request by dpanta94**
Providing store data from php to the our redux store

### CODE REVIEW

File: src/Tickets/Commerce/Order_Modifiers/API/Fees.php [hunk 1]
```diff
@@ -377,60 +419,4 @@ protected function update_fees_for_ticket( $ticket_id, $fees ) {
...
 	}
-
-	/**
-	 * Get the callback for adding fees to the ticket data.
-	 *
-	 * @since TBD
-	 *
-	 * @return callable The callback for adding fees to the ticket data.
-	 */
-	protected function get_single_ticket_data_callback(): callable {
```
Comment (JPry) [inline src/Tickets/Commerce/Order_Modifiers/API/Fees.php:388]: We don't need to remove these, they work well as-is and allow us to wrap the method calls as we need them.

**Review by JPry — CHANGES_REQUESTED** [review]

File: src/Tickets/Commerce/Order_Modifiers/API/Fees.php [hunk 2]
```diff
@@ -313,19 +354,20 @@ protected function update_fees_for_ticket_response( Request $request ) {
...
 	 *
+	 * @param int   $post_id     The post ID.
 	 * @param int   $ticket_id   The ticket ID.
 	 * @param array $ticket_data The ticket data.
 	 *
 	 * @return void
 	 */
-	protected function save_fees_for_ticket( $ticket_id, $ticket_data ) {
-		if ( ! isset( $ticket_data['fees']['selected_fees'] ) ) {
+	public function save_fees_for_ticket( $post_id, $ticket_id, $ticket_data ) {
```
[the remark this answers is not in the document]
Comment (dpanta94) [reply src/Tickets/Commerce/Order_Modifiers/API/Fees.php:363]: Im confident we will reach the place where we will need the $post_id. Regarding having wrappers for callbacks, we discussed it already and i disagree with your logic. I think they dont offer anything other than over engineering and complications.

File: src/Tickets/Commerce/Order_Modifiers/API/Fees.php (hunk 1 above)
Comment (dpanta94) [reply src/Tickets/Commerce/Order_Modifiers/API/Fees.php:388]: responded above.

File: src/Tickets/Commerce/Order_Modifiers/API/Fees.php [hunk 3]
```diff
@@ -255,6 +258,44 @@ protected function get_fees_for_ticket( int $ticket_id ) {
...
+	public function get_selected_fees_for_post_by_ticket( int $post_id ) {
+		$provider = Tickets::get_event_ticket_provider( $post_id );
+
+		if ( Module::class !== $provider ) {
+			return [];
+		}
+
+		$fees_for_post_by_ticket = [];
+
+		foreach ( tribe_tickets()->where( 'event', $post_id )->get_ids( true ) as $ticket_id ) {
```
[the remark this answers is not in the document]
Comment (dpanta94) [reply src/Tickets/Commerce/Order_Modifiers/API/Fees.php:292]: The function is actually better. It passes the repository through filters which are being utilized by our plugins.. Thats the safe way to use it.

File: src/Tickets/Commerce/Order_Modifiers/API/Fees.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (dpanta94) [reply src/Tickets/Commerce/Order_Modifiers/API/Fees.php:363]: If we have a reason to do it like that we could use our container's callback method. But again i dont see the reason. Sorry.

**Pull request merged**